codehs.user.css

```
@@ -1,10 +1,10 @@
 /* ==UserStyle==
 @name CodeHS Catppuccin
 @namespace https://github.com/qDread-dev/codehs-cat
-@version 0.0.27
+@version 0.0.28
 @updateURL https://raw.githubusercontent.com/qDread-dev/codehs-cat/refs/heads/main/codehs.user.css
 @description Soothing pastel theme for CodeHS
```

```suggestion
@description Soothing pastel theme for CodeHS. By qdread-dev and TEEJIO
```

add credit in description